### ISSUE TITLE
Add alt text to all shown image resources

### ISSUE THREAD

**Issue by Taewan-P** [BusHanyang/hybus-genesis#68]
For better accessibility, all images should have alt text according to the lighthouse report.

Therefore, we should check whether all icon resources have alt text in their property.
